examples-standalone/kendoangular-java-integration/readme.md

```diff
@@ -0,0 +1,93 @@
+# Kendo UI for Angular Grid and Upload Component with Java Spring Boot: A Sample Project
```

Comment:
how about shortening the title a bit to `Kendo UI for Angular with Java Spring Boot Integration`?

in the description, we can add details about the Grid and Upload.